warehouse/integrations/postgres/load.go

```diff
@@ -143,7 +130,7 @@ func (pg *Postgres) loadTable(
 	}
 
 	var rowsDeleted int64
-	if !slices.Contains(pg.config.skipDedupDestinationIDs, pg.Warehouse.Destination.ID) {
+	if isUsersTable || pg.shouldMerge() {
```

Comment:
Since we already have a tableName in here, we can decide based on that? wdyt?
```suggestion
	if tableName == warehouseutils.UsersTable || pg.shouldMerge() {
```

Reply:
Yeah I thought about that, but it felt more robust to have the caller tell  the `loadTable` what behaviour is needed. 

That's because the `loadTable` method seems to be unaware of the concept of "users", it doesn't care about the data, it just needs to load the table.

We could even rename `isUsersTable` to `forceMerge` and remove all references of users from `loadTable`. Thoughts?

Reply:
`forceMerge` seems to be a good option.